### PR TITLE
Fix resizing in PReLU layer

### PR DESCRIPTION
**Pull request by kostinalexey**

### CODE REVIEW

**Comment by rajeevsrao** [comment]
@pranavm-nvidia can you please review?

File: parsers/caffe/caffeParser/opParsers/parsePReLU.cpp [hunk 1]
```diff
@@ -37,10 +37,26 @@ ILayer* parsePReLU(INetworkDefinition& network, const trtcaffe::LayerParameter&
...
-    slopesDims.d[1] = nWeights;
+    int nWeights = channelShared ? 1 : inputDims.d[0];
 
+    Dims slopesDims{inputDims.nbDims, {nWeights, 1, 1, 1, 1, 1, 1, 1},
+                    {DimensionType::kCHANNEL, DimensionType::kSPATIAL, DimensionType::kSPATIAL, DimensionType::kSPATIAL,
+                     DimensionType::kSPATIAL, DimensionType::kSPATIAL, DimensionType::kSPATIAL, DimensionType::kSPATIAL}};
+    for(int i = 0; i < inputDims.nbDims; i++) {
+        int d{0};
+        switch(inputDims.type[i]) {
+        case DimensionType::kSPATIAL:
```
Comment (pranavm-nvidia) [inline parsers/caffe/caffeParser/opParsers/parsePReLU.cpp:48]: `DimensionType` is deprecated, so I don't think we should rely on it here.

Comment (kostinalexey) [reply parsers/caffe/caffeParser/opParsers/parsePReLU.cpp:48]: Existing version has 2 bugs:
1. does not initialise all dimensions here
`Dims slopesDims{inputDims.nbDims, {1}, {DimensionType::kSPATIAL}};`
2. does not work with dynamic reshape.

If it is not possible to use `DimensionType` what is the alternative?

Comment (nvpohanh) [reply parsers/caffe/caffeParser/opParsers/parsePReLU.cpp:48]: The `DimensionType` field has been removed in TRT 8. Maybe we should fix this in the latest branch instead?

**Comment by nvpohanh** [comment]
We are deprecating Caffe parser in favor of ONNX. Close this for now.

**Pull request closed without merging**